### PR TITLE
glossarium:0.1.0

### PR DESCRIPTION
**Pull request by slashformotion**
I am submitting
- [x] a new package
- [ ] an update for a package

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

Glossarium is a simple, easily customizable typst glossary inspired by [LaTeX glossaries package](https://www.ctan.org/pkg/glossaries) 
image

### CODE REVIEW

File: packages/preview/glossarium-0.1.0/typst.toml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[package]
+name = "glossarium"
+version = "0.1.0"
+entrypoint = "glossarium.typ"
+authors = ["slashformotion", "Dherse"]
+license = "MIT"
+description = "Glossarium is a simple, easily customizable typst glossary."
+repository = "https://github.com/slashformotion/glossarium"
+exclude = ["LICENCE", "example/*"]
```
Comment (laurmaedje) [inline packages/preview/glossarium-0.1.0/typst.toml:9]: I don't recommend excluding the license. It doesn't increase the package size by much and it is better if it always shipped with the code.

File: packages/preview/glossarium-0.1.0/README.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+For Typst before 0.6.0 or to use **glossarium** as a local module, download the package files into your project folder and import `glossarium.typ`:
+
+```js
+#import "glossarium.typ": make-glossary, print-glossary, gls, glspl 
+```
+
+After importing the package and before making any calls to `gls`,` print-glossary` or `glspl`, please ***MAKE SURE*** you add this line
+```js
+#show: make-glossary.with()
```
Comment (laurmaedje) [inline packages/preview/glossarium-0.1.0/README.md:29]: The `.with()` shouldn't have any effect without arguments.
```suggestion
#show: make-glossary
```

File: packages/preview/glossarium-0.1.0/README.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+A term is a [dictionary](https://typst.app/docs/reference/types/dictionary/) composed of 2 required and 2 optional elements: 
+
+- `key` (string) *required*: used to make reference to the term.
+- `short` (string) *required*: the short form of the term replacing the term citation. 
+- `long` (string or content) *optional*: The long form of the term, displayed in the glossary and on the first citation of the term. 
+- `desc` (string or content) *optional*: The description of the term.
+
+Then the terms are passed as a list to `print-glossary`
+```ts
+#show: print-glossary.with((
```
Comment (laurmaedje) [inline packages/preview/glossarium-0.1.0/README.md:49]: Why does this require a show rule? Does it affect what comes after?

File: packages/preview/glossarium-0.1.0/README.md (hunk 2 above)
Comment (slashformotion) [reply packages/preview/glossarium-0.1.0/README.md]: changed

File: packages/preview/glossarium-0.1.0/README.md (hunk 3 above)
Comment (slashformotion) [reply packages/preview/glossarium-0.1.0/README.md]: changed

File: packages/preview/glossarium-0.1.0/typst.toml (hunk 1 above)
Comment (slashformotion) [reply packages/preview/glossarium-0.1.0/typst.toml]: changed

**Comment by laurmaedje** [comment]
Thank you!

**Pull request merged**